layers/+tools/tide/funcs.el

```diff
@@ -0,0 +1,85 @@
+;;; funcs.el --- Tide  Layer functions File for Spacemacs
+;;
+;; Copyright (c) 2012-2020 Sylvain Benner & Contributors
+;;
+;; Author: Thanh Vuong <thanhvg@gmail.com>
+;; URL: https://github.com/thanhvg
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defun spacemacs//tide-setup-prefix ()
+  "This one should run only once, otherwise `which-key' will become very slow #12455"
+  (dolist (mode tide-managed-modes)
+    (spacemacs/declare-prefix-for-mode mode "mE" "errors")
+    (spacemacs/declare-prefix-for-mode mode "mg" "goto")
+    (spacemacs/declare-prefix-for-mode mode "mh" "help")
+    (spacemacs/declare-prefix-for-mode mode "mn" "name")
+    (spacemacs/declare-prefix-for-mode mode "mr" "refactor")
+    (spacemacs/declare-prefix-for-mode mode "mS" "server")))
+
+(defun spacemacs//tide-setup-bindings ()
+  "Define keys bindings for `tide-mode'"
+  (spacemacs/set-leader-keys-for-minor-mode 'tide-mode
```

Comment:
Are you sure that binding these is enough for the minor mode only? I would have assumed these would need to be bind for the major modes instead.

Reply:
Yeah I think it's ok, this how `lsp` layer adds its key bnidings